eliot/tests/strategies.py

```diff
@@ -87,7 +98,8 @@ def union(*dicts):
 start_action_message_dicts = builds(
     union, message_dicts, _start_action_fields).map(
         lambda x: x.update({TASK_LEVEL_FIELD: x[TASK_LEVEL_FIELD].set(-1, 1)}))
-start_action_messages = start_action_message_dicts.map(WrittenMessage.from_dict)
+start_action_messages = start_action_message_dicts.map(thaw).map(
```

Comment:
I think this .map(thaw) is unneeded given that written_from_pmap also thaw's its argument.